portable_fs/template/fs.py

```diff
@@ -11,6 +11,7 @@
 from . import core
 from collections import defaultdict
 import warnings
+import pathlib
```

Comment:
That's a reasonable assumption. As you know these are toy projects, which is why CI testing is not rigorously set up and some py3-only code slipped through. We're moving all these into the broker repo soon-ish, so the content of this PR will need to be refactored to be py2 compatible either now or later. Better to do it now so it's not one more test failure to fix later.